application/controllers/CommentsController.php

```diff
@@ -161,11 +161,13 @@ public function detailsAction()
 
         $this->addControl((new CommentList($rs))->setViewMode('minimal'));
 
-        $this->addControl(new ShowMore(
-            $rs,
-            Links::comments()->setQueryString(QueryString::render($this->getFilter())),
-            sprintf(t('Show all %d comments'), $comments->count())
-        ));
+        if ($comments->count() > 3) {
```

Comment:
Why do you need this? The `ShowMore` widget does that internally already.

Reply:
Without this, an empty show-more container will be rendered, which takes up unecessary space. 

### Current Icinga DB Web Master

![Screen Shot 2021-06-02 at 09 57 08](https://user-images.githubusercontent.com/6977434/120445483-f21eb780-c388-11eb-921a-e075c06b4306.png)

Reply:
Then please play with `:empty` or so.

Reply:
Or override `renderUnwrapped()` in the `ShowMore` widget and perform the same check as in `assemble` to return an empty string if successful.

Reply:
✅